modules/ROOT/pages/portal-vanity-domain.adoc

```diff
@@ -65,11 +65,11 @@ For example:
 
 Add headers for each route described above.
 
-[source,console,linenums]
-----
-'X-Forwarded-Path': Base path of your vanity domain public portal URL. This can be any path. It can be just '/' for the root path. It does not need to match the base path for the portal in the anypoint.mulesoft.com domain. This path must end with a slash ("/"). Example: 'path/of/justice/'
-'X-Forwarded-Host': Your public portal vanity domain. Example: 'developers.robocop.com'
-----
+* `X-Forwarded-Path` is the base path of your vanity domain public portal URL, such as `/path/of/justice/`.
++
+It can be any path that begins and ends with a slash ("/"), and it can be just `/` for the root path. It does not need to match the base path for the portal in the anypoint.mulesoft.com domain.
```

Comment:
```suggestion
The vanity domain public portal base path can be either any path that begins and ends with a slash (`/`) or a root path represented by a single slash (`/`). It does not need to match the base path for the anypoint.mulesoft.com domain portal.
```
Please carefully reread to be sure that my suggested rewording did not change technical meaning.

Reply:
I rewrote this for better clarity and consistency and incorporated your suggestions. Thank you!